components/automate-cli/cmd/chef-automate/cleanup.go

```diff
@@ -59,46 +75,46 @@ func runCleanupCmd(cmd *cobra.Command, args []string) error {
 				opensearchIps := infra.Outputs.OpensearchPrivateIps.Value
 				for i := 0; i < len(automateIps); i++ {
 					servername := "Automate"
-					writer.Println("cleanup is starting on " + servername + " node : " + automateIps[i]+ "\n")
+					writer.Println("Cleanup has started on " + servername + " node : " + automateIps[i] + "\n")
 					_, err := ConnectAndExecuteCommandOnRemote(sshUser, sshPort, sskKeyFile, automateIps[i], FRONTENDCLEANUP_COMMANDS)
 					if err != nil {
 						writer.Errorf("%s", err.Error())
 						return err
 					} else {
-						writer.Success("cleanup is successfully completed on " + servername + " node : " + automateIps[i] + "\n")
+						writer.Success("Cleanup is completed on " + servername + " node : " + automateIps[i] + "\n")
 					}
 				}
 				for i := 0; i < len(chefserverIps); i++ {
 					servername := "chef server"
-					writer.Println("cleanup is starting on " + servername + " node : " + chefserverIps[i]+ "\n")
+					writer.Println("Cleanup has started on " + servername + " node : " + chefserverIps[i] + "\n")
 					_, err := ConnectAndExecuteCommandOnRemote(sshUser, sshPort, sskKeyFile, chefserverIps[i], FRONTENDCLEANUP_COMMANDS)
 					if err != nil {
 						writer.Error(err.Error())
 						return err
 					} else {
-						writer.Success("cleanup is successful on " + servername + " node : " + chefserverIps[i] + "\n")
+						writer.Success("Cleanup is completed on " + servername + " node : " + chefserverIps[i] + "\n")
 					}
 				}
```

Comment:
we can create func like backendpostgrescleanup()

Reply:
Outdated